### PR TITLE
[INLONG-4812][Manager] Choose the specified connector instead of all

### LINKED ISSUE #4812: [Improve][Manager] Choose the specified connector instead of all
### Description

Choose the specified connector instead of all to avoid conficts

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by LvJiancheng**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example:[INLONG-4812][Manager] Choose the specified connector instead of all

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #4812 

### Motivation

Choose the specified connector instead of all

### Modifications

Choose the specified connector instead of all

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

**Review by woofyzhao — APPROVED** [review]

File: inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/FlinkOperation.java [hunk 1]
```diff
@@ -161,15 +174,46 @@ public void genPath(FlinkInfo flinkInfo, String dataflow) throws Exception {
         flinkInfo.setLocalJarPath(jarPath);
         log.info("get sort jar path success, path: {}", jarPath);
 
+        List<String> nodeTypes = new ArrayList<>();
+        if (StringUtils.isNotEmpty(dataflow)) {
+            JsonNode streams = JsonUtils.parseTree(dataflow).get(InlongConstants.STREAMS);
+            for (int i = 0; i < streams.size(); i++) {
+                JsonNode relations = streams.get(i).get(InlongConstants.RELATIONS);
+                for (int j = 0; j < relations.size(); j++) {
+                    String inputNames = OBJECT_MAPPER.convertValue(relations.get(j).get(InlongConstants.INPUTS),
```
Comment (healchow) [inline inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/FlinkOperation.java:183]: Why name them `inputNames` and `outputNames`? Should remove the last char `s`?

File: inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/FlinkOperation.java [hunk 2]
```diff
@@ -161,15 +174,46 @@ public void genPath(FlinkInfo flinkInfo, String dataflow) throws Exception {
...
+                                    new TypeReference<List<String>>(){}).stream().findFirst().orElse(null);
+                    String outputNames = OBJECT_MAPPER.convertValue(relations.get(j).get(InlongConstants.OUTPUTS),
+                            new TypeReference<List<String>>(){}).stream().findFirst().orElse(null);
+
+                    if (inputNames.equals(outputNames)) {
+                        String message = String.format("input nodeName: %s equals to output nodeName: %s",
```
Comment (healchow) [inline inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/FlinkOperation.java:189]: Suggest more clearly, for example, "input nodeName %s cannot be equal to output nodeName %s".

Comment (LvJiancheng) [reply inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/FlinkOperation.java:189]: ok

File: inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/enums/ConnectorJarType.java [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.plugin.flink.enums;
+
+import lombok.Getter;
+
+@Getter
+public enum ConnectorJarType {
```
Comment (yunqingmoswu) [inline inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/enums/ConnectorJarType.java:23]: Please add some comment.

File: inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/enums/ConnectorJarType.java [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+    DLCICEBERG_SINK("dlcIcebergLoad", "dlc"),
+
+    HIVE_SINK("hiveLoad", "hive"),
+
+    ICEBERG_SINK("icebergLoad", "iceberg"),
+
+    HDFS_SINK("fileSystemLoad", ""),
+
+    ;
```
Comment (yunqingmoswu) [inline inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/enums/ConnectorJarType.java:71]: Maybe it is better to format it.

**Review by kipshi — APPROVED** [review]

**Review by yunqingmoswu — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by healchow — APPROVED** [review]
LGTM

**Pull request merged**